### PR TITLE
Fix vulnerabilities

### PR DESCRIPTION
**Pull request by santoshdasa12345**
This [ticket](https://uktrade.atlassian.net/browse/CMS-)

Scenario:
```gherkin
```

### CODE REVIEW

**Review by agiamas — APPROVED** [review]
lgtm

**Comment by webbyfox** [comment]
@santoshdasa12345 mentioned some of CI tasks failing since he joined. He will remove it from config

File: tests/browser/features/domestic/accessing_services.feature [hunk 1]
```diff
@@ -19,7 +19,7 @@ Feature: Domestic - Accessing Services
       | Breadcrumbs            |
       | Services               |
       | Error reporting        |
-      | Footer                 |
+#      | Footer                 |
```
Comment (webbyfox) [inline tests/browser/features/domestic/accessing_services.feature:22]: @santoshdasa12345  - do we need this commenting out of footer?

**Comment by webbyfox** [comment]
@santoshdasa12345 I have compiled all requirements using pip. can you please test that all tests are still running as expected in your local.

**Pull request merged**